File: pertemuan1/atm01.js
let saldo = parseFloat(prompt("Masukkan saldo: "));
let penarikan = parseFloat(prompt("Masukkan jumlah penarikan: "));

if (penarikan <= saldo) {
  saldo -= penarikan;
  console.log(`Transaksi berhasil! Sisa saldo: ${saldo.toFixed(2)}`);
} else {
  console.log("Saldo tidak cukup.");
}

File: pertemuan1/atm01.py
saldo = float(input("Masukkan saldo: "))
penarikan = float(input("Masukkan jumlah penarikan: "))
  
if penarikan <= saldo:
    saldo -= penarikan
    print(f"Transaksi berhasil! Sisa saldo: {saldo:.2f}")
else:
    print("Saldo tidak cukup.") 